test/test-utils/helper/src/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { pairs, every } = require('underscore');
+
+const insufficientSetUp = (env) => {
```

Comment:
Keep method names consistent. Rename this method to `insufficientSetup` (lowercase `u`) so that it is consistent with the exposed method.